idea_town/frontend/static-src/app/templates/scroll-header-view.js

```diff
@@ -8,7 +8,12 @@ export default `
         </header>
         {{^scrolled}}
           <div id="avatar-wrapper">
-            <span class="default-avatar" data-hook="logout"></span>
+            {{#avatar}}
+              <img class="avatar" src="{{avatar}}" width="41" height="41" data-hook="logout">
```

Comment:
Should we just set `context.avatar = 'static/default-avatar.png'`to avoid the mustache templating here, and simplify the template? re: our conversation yesterday.